### ISSUE TITLE
How to set the default path for PDF files

### ISSUE THREAD

**Issue by thunderbolt-fire** [freddyaboulton/gradio-pdf#12]
```
import gradio as gr
from gradio_pdf import PDF

with gr.Blocks() as demo:
    pdf = PDF(label="Upload a PDF", value='/workspaces/LLM-RAG-Gradio/python/pythonTrainingData_w3schools.pdf',min_width="500%")
    
    name = gr.Textbox()
    pdf.upload(lambda f: f, pdf, name)

demo.launch()
```

I set `value` the path,but don't work.

**Comment by freddyaboulton** [comment]
My mistake - should have a fix out momentarily.

**Comment by freddyaboulton** [comment]
Fixed in version 0.0.12
